src/Components/AddClassOnline/AddClassOnline.tsx

```diff
@@ -0,0 +1,35 @@
+import styles from "./AddClassOnline.module.css";
+import Clock from "./assets/clock.svg";
+import Location from "./assets/location.svg";
+import Plus from "./assets/plus.svg";
```

Comment:
It's just a small thing, but maybe it's more clear to use lowercase names for the image sources so you don't think they are React components.